spec/services/hashtag_autocomplete_service_spec.rb

```diff
@@ -39,19 +39,19 @@
   describe "#search" do
     it "returns search results for tags and categories by default" do
       expect(subject.search("book", %w[category tag]).map(&:text)).to eq(
-        ["The Book Club", "great-books x 22"],
```

Comment:
I believe all these tests should check `secondary_text` too.

Reply:
@martin-brennan follow up in another commit?